modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Checkbox/Checkbox.tsx

```diff
@@ -34,29 +34,21 @@ const Switcher: React.FC<ISwitcherProps> = ({
 	return (
 		<>
 			<label className="ddm-toggle-switch toggle-switch">
```

Comment:
Remove all unused CSS classes
- Maybe the `ddm-toggle-switch` class is no necessary anymore, since the Clay probably contains all the style need.
```suggestion
			<label className="toggle-switch">
```